spec/factories/valid_collection_draft_proposal_factories.rb

```diff
@@ -0,0 +1,893 @@
+require 'faker'
```

Comment:
https://github.com/nasa/mmt/pull/382/files#diff-a36a537bfe0e4dbbc278308dc21778cd
https://github.com/nasa/mmt/pull/382/files#diff-04aebb6b804325ff69280a86ac31c27f

Reply:
In summary, I stuffed all of the reusable chunks of the factory into a function and then merged anything that we might want to change in afterwards.